accounting/docs.md

```diff
@@ -0,0 +1,43 @@
+# Accounting Reports
+Faraday produces accounting reports on a node's on chain and off chain activity. 
+These reports are formatted using the [Harmony Reporting Standard](https://github.com/picksco/harmony). 
+This document provides a description of the entries in these reports. 
+
+## Common Fields
+For brevity, the following fields which have the same meaning for each entry will be omitted: 
+- Timestamp: The timestamp of the block that the channel open transaction appeared in. 
+- Fiat: The value of the amount field in USD. Note that values less than one satoshi will be rounded down to zero. 
```

Comment:
So, say if I only ever transact less then 1 sat transactions (say with zero fee), then the sum would be zero?

Reply:
As is yes, but would make a change in the fiat package to quote for sub-satoshi values as well. I think the accountant's perspective on this is that we can't exchange these tiny amounts for fiat, so they have an effective 0 value. 

But now that you point it out, maybe better to quote and let people round down themselves? Particularly if we're writing accounting for lightning.

Reply:
Over time these sub satoshi transactions can add up, so in the end it's better to quote imo.